relayer/src/keyring.rs

```diff
@@ -389,6 +371,33 @@ impl KeyRing {
     }
 }
 
+/// Sign a message
+pub fn sign_message(
+    key: &KeyEntry,
+    msg: Vec<u8>,
+    address_type: &AddressType,
+) -> Result<Vec<u8>, Error> {
```

Comment:
This is refactored from:

https://github.com/informalsystems/ibc-rs/blob/3e34b4b916eb91f680412e5fdd3ea045a99526d2/relayer/src/keyring.rs#L355-L360